### PR TITLE
Presentation API test

### PR DESCRIPTION
**Pull request by louaybassbouss**
Dear all,

I am the Test Facilitator of the [W3C Presentation API](http://www.w3.org/TR/presentation-api/). This PR contains the initial folder structure and idlharness of the API for [controlling](http://www.w3.org/TR/2015/WD-presentation-api-20151013/#dfn-controlling-user-agent) and [receiving](http://www.w3.org/TR/2015/WD-presentation-api-20151013/#dfn-receiving-user-agent) user agents.  I am happy to hear your feedback. 

Thx,
Louay

### CODE REVIEW

**Comment by tobie** [comment]
Reviewers for this pull request are: @w3c/presentation-api-reviewers.

**Comment by zqzhang** [comment]
w3c-test:mirror

**Comment by zqzhang** [comment]
Feel free to try the tests at
http://w3c-test.org/submissions/2432/presentation-api/

**Comment by mfoltzgoogle** [comment]
Thanks for kicking this off @louaybassbouss.  Tests LGTM

File: presentation-api/controlling-ua/idlharness.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    "blob",
+    "arraybuffer"
+};
+
+interface PresentationConnection : EventTarget {
+    readonly attribute DOMString? id;
+    readonly attribute PresentationConnectionState state;
+    void close();
+    void terminate();
+    attribute EventHandler onstatechange;
```
Comment (zqzhang) [inline presentation-api/controlling-ua/idlharness.html]: Hmm, seems the spec has some changes at https://github.com/w3c/presentation-api/pull/227

Would you please update the IDL accordingly? Or shall we merge this as-is and address IDL update in following PRs?

Maybe you can add a comment in the test file to reflect the spec date for tracking.

**Comment by zqzhang** [comment]
LGTM with nit.

**Comment by zqzhang** [comment]
As @louaybassbouss committed to keep the IDL up-to-date with the spec at https://github.com/w3c/presentation-api/issues/266#issuecomment-184208099, I would like to land the initial pull request as-is.

**Pull request merged**